.github/workflows/continous-delivery.yml

```diff
@@ -23,6 +26,32 @@ env:
   MACHINE_TYPE: c2-standard-4
 
 jobs:
+  # If a release was made we want to substract the first part of the semver from the
+  # tag_name
+  #
+  # Output:
+  # - If our semver is v1.3.0 the resulting output from this job would be v1
+  # 
+  # Note: We just use the first part of the version to replace old instances, and change
+  # it when a major version is released, to keep a segregation between new and old 
+  # versions.
+  versioning:
+    name: Versioning
+    runs-on: ubuntu-latest
+    outputs:
+      api_version: ${{ steps.set.outputs.api_version }}
```

Comment:
On semver.org, this is called the "major" version, and that's the convention for Rust projects.

Reply:
Fixed by [fix(build): use rust conventions for versioning](https://github.com/ZcashFoundation/zebra/pull/4939/commits/98c9b87d96071bf12307b41f3a1a6fdf36804a75)